### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ppanero**
Improvements on installation:
-mibox
-pydeep
-pylief

Configuration for version 2.4.83 and 2.4.84 added

### CODE REVIEW

**Comment by ppanero** [comment]
@bastelfreak  could you merge (dont want to merge my own) and generate a new release? its been tested in our instances and we would like to use the latest changes in an upcoming workshop this week.

File: manifests/dependencies.pp [hunk 1]
```diff
@@ -5,14 +5,39 @@
...
+    dev     => 'present',
+  }
+
+  if $misp::pymisp_rpm {
+    ensure_packages( ['pymisp'],
+      { 'ensure' => 'present' }
+    )
+  }
+  else {
+    ::python::pip { 'pymisp' :
```
Comment (ekohl) [inline manifests/dependencies.pp]: The leading colons aren't needed.

File: manifests/dependencies.pp [hunk 2]
```diff
@@ -5,14 +5,39 @@
...
+      pkgname => 'pymisp',
+    }
+  }
+
+  if $misp::lief {
+    ensure_packages( [$misp::lief_package_name],
+      { 'ensure' => 'present' }
+    )
+  }
+}
```
Comment (ekohl) [inline manifests/dependencies.pp]: In general we have newlines at the end of the file

File: manifests/dependencies.pp [hunk 3]
```diff
@@ -5,14 +5,39 @@
...
+    version => 'system',
+    pip     => 'present',
+    dev     => 'present',
+  }
+
+  if $misp::pymisp_rpm {
+    ensure_packages( ['pymisp'],
+      { 'ensure' => 'present' }
+    )
+  }
```
Comment (ekohl) [inline manifests/dependencies.pp]: Could you add the else to this line?

File: manifests/install.pp [hunk 4]
```diff
@@ -73,6 +81,15 @@
     subscribe   => Vcsrepo["${misp::install_dir}/app/files/scripts/mixbox"],
   }
 
+  exec {'pydeep build':
+    command     => '/usr/bin/python setup.py build && /usr/bin/python setup.py install',
+    cwd         => "${misp::install_dir}/app/files/scripts/pydeep/",
+    unless      => '/usr/bin/pip list | grep pydeep',
```
Comment (ekohl) [inline manifests/install.pp]: Won't this prevent upgrades? Currently it's refreshonly and subscribing to the vcsrepo so I'd expect it's already idemptotent.

**Comment by alexjfisher** [comment]
The PR title will end up being used in the changelog.
@ppanero Can you update it to something other than just 'Features'?

**Comment by alexjfisher** [comment]
Instead of merging from upstream, we prefer PRs to be rebased.

**Comment by ppanero** [comment]
@alexjfisher  I will take care of rebasing and making the appropriate changes this week. Sorry for the delay

**Comment by bastelfreak** [comment]
Hey @ppanero. I'm going to close this PR due to inactivity. Please reopen and rebase it if you're stll interested in getting this merged.

**Pull request closed without merging**